packages/modules-sdk/src/definitions.ts

```diff
@@ -74,7 +74,7 @@ export const ModulesDefinition: { [key: string]: ModuleDefinition } = {
     isRequired: false,
     canOverride: true,
     isQueryable: true,
-    dependencies: [],
+    dependencies: ["eventBusService"],
```

Comment:
todo: This is the eventBusService from the core, so `IEventBusService` I can see that everywhere you are using `IEventBusModuleService`